modules/pool/pool.gql

```diff
@@ -1502,7 +1502,26 @@ interface GqlPoolEvent {
     userAddress: String!
     poolId: String!
     timestamp: Int!
+    """
+    USD amount related to the event
+    """
     valueUSD: Float!
+    """
+    Swap input token
+    """
+    tokenIn: GqlPoolEventAmount
+    """
+    Swap output token
+    """
+    tokenOut: GqlPoolEventAmount
+    """
+    Surplus generate by CowAmm swap
+    """
+    surplus: GqlPoolEventAmount
```

Comment:
should we also acc `fee: GqlPoolEventAmount` to make it complete?

Reply:
good idea

Reply:
added